docs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "strong-xstats",
```

Comment:
AFAIK, the only naming convention we have is to use `strong-` prefix. I am also not a big fan of long module names created as a concatenation of all relevant keywords, that's why I was trying to come up with a shorter name. Not a big deal, if you think a long descriptive name is better, then I can live with that.

The ultimate goal is to collect "API usage metrics", so perhaps the name can include some part from that?

Few candidates:

 - strong-api-stats
 - strong-express-agent
 - strong-express-stats
 - strong-express-metrics
 - strong-api-metrics
 - strong-api-agent

Thoughts?